miniconda/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER Travis Swicegood
 
 RUN apt-get update && apt-get install -y wget bzip2
 RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
-    wget --quiet http://repo.continuum.io/miniconda/Miniconda-3.9.1-Linux-x86_64.sh && \
+    wget --quiet https://repo.continuum.io/miniconda/Miniconda-3.9.1-Linux-x86_64.sh && \
```

Comment:
I'm getting exit code 5 on this line for some reason.  Are you seeing any issues with this, @rgbkrk?